src/app/pages/logs/list/logs-list-table-data-source.ts

```diff
@@ -40,11 +40,18 @@ export class LogsListTableDataSource extends TableDataSource<Log> {
     private authorizationService: AuthorizationService,
     private router: Router,
     private centralServerNotificationService: CentralServerNotificationService,
+    private activatedRoute: ActivatedRoute,
     private centralServerService: CentralServerService,
     private datePipe: AppDatePipe) {
     super(spinnerService, translateService);
     // Init
     this.initDataSource();
+    this.activatedRoute.queryParams.subscribe(params => {
+      this.setSearchValue(params['id'] ? params['id'] : '');
+      if (params['fromChargingProfile']) {
+        this.setStaticFilters([{ Action: 'ChargingProfiles|ChargingProfileDelete|ChargingProfileUpdate' }]);
```

Comment:
When a static filter is set, it is not possible to filter for other actions properly anymore